lxml-stubs/builder.pyi

```diff
@@ -0,0 +1,30 @@
+from typing import Any, Callable, Mapping, Optional
+
+from .etree import _AnyStr, _Element
+
+class ElementMaker:
+    def __init__(
+        self,
+        typemap: Optional[Mapping[Any, Callable[[_Element, Any], None]]] = ...,
+        namespace: Optional[str] = ...,
+        nsmap: Optional[Mapping[Any, _AnyStr]] = ...,
+        # same signature as etree.Element()
+        makeelement: Optional[Callable[..., _Element]] = ...,
+    ) -> None: ...
+    def __call__(
+        self,
+        tag: str,

```

Comment:
Interesting. It doesn't currently accept `QName`s, mostly due to unfortunate implementation details. That should be fixed.

Reply:
Cool, I can submit another PR once the fix has landed.